src/client/App.jsx

```diff
@@ -8,22 +8,92 @@ const apiUrl = 'http://localhost:4000';
 function App() {
   const [movies, setMovies] = useState([]);
 
+
   useEffect(() => {
-    fetch(`${apiUrl}/movie`)
+    const options = {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        'Authorization': 'Bearer ' + localStorage.getItem('jwt')
+      }
+    }
+    fetch(`${apiUrl}/movie`, options)
       .then(res => res.json())
-      .then(res => setMovies(res.data));
+      .then(res => {
+        console.log(res.data)
+        setMovies(res.data)
+      });
   }, []);
 
   const handleRegister = async ({ username, password }) => {
-    
+
+    const options = {
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json',
+      },
+      body: JSON.stringify({
+        username: username,
+        password: password
+      }),
+    }
+    fetch(`${apiUrl}/user/register`, options)
+    .then(res => res.json())
+    .then(json => console.log('Registered user:' + json.data.username))
+    .catch(() => console.log('Failed to register user'))
   };
 
   const handleLogin = async ({ username, password }) => {
     
+    const options = {
```

Comment:
having the options outside the fetch makes it so much easier to read, and something I didnt really consider before. Will do that in future